trunk/SUAVE/Analyses/Aerodynamics/Fidelity_Zero.py

```diff
@@ -92,7 +92,6 @@ def __defaults__(self):
         compute.drag.parasite.propulsors.propulsor = Common.Drag.parasite_drag_propulsor
         compute.drag.parasite.pylons               = Common.Drag.parasite_drag_pylon
         compute.drag.parasite.total                = Common.Drag.parasite_total
-        compute.drag.induced                       = Common.Drag.induced_drag_aircraft

```

Comment:
Why was this removed?

Reply:
The induced drag is computed by the VLM. This function is not of use for fidelity zero anymore.

Reply:
I don't we should remove it. Getting a reasonable value from the VLM requires extra geometry definition (i.e. twist) that may not be available.